com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -409,10 +409,27 @@ public IReadOnlyList<ulong> ConnectedClientsIds
         public event Action<ulong> OnClientDisconnectCallback = null;
 
         /// <summary>
-        /// The callback to invoke once the server is ready
+        /// The callback to invoke once the local server is ready
         /// </summary>
         public event Action OnServerStarted = null;
 
+        /// <summary>
+        /// The callback to invoke once the local client is ready
```

Comment:
Are you sure about the "not yet connected" part? When is that connection happening? I see that             `MessagingSystem.ClientConnected(ServerClientId);` is called before this, but maybe you mean something else

Reply:
Oh wait, I think what you mean is that, in host mode, this is called after `InvokeOnClientConnectedCallback`? If yes, I'd change the code I added so that the order of things is consistent regardless of if you're running in client only vs host mode.

What are your thoughts on this?

Reply:
That too works and then you don't need the remarks about the host client already being connected.
👍

Reply:
perfect, done